netci.groovy

```diff
@@ -345,6 +345,61 @@ def targetGroupOsMapInnerloop = ['netcoreapp': ['Windows_NT', 'Ubuntu14.04', 'Ub
     } // targetGroup
 } // isPR
 
+// **************************
+// Define Linux ARM64 cross builds. These jobs run on every merge.
```

Comment:
Instead of duplicating this whole section, would it make more sense to refactor it into the above section: "Define Linux ARM builds."?

Reply:
This was drafted from the simpler x86 section below.  The arm version has more complications with its multiple flavors

Reply:
@eerhardt 
There are subtle differences between x86, arm and arm64 which makes it complicated to merge these.
My groovy skills are immature at the moment.  I would prefer to crawl before trying to walk.

If you disagree I can take a look tomorrow

Reply:
ok, that's fine with me.